druid/examples/web/build.rs

```diff
@@ -122,21 +122,15 @@ mod examples {
         if let Some(example) = path.file_stem() {
             let example_str = example.to_string_lossy();
 
-            // Skip examples that are known to not work with wasm.
+            // Skip examples that are known to not work.
             if EXCEPTIONS.contains(&example_str.as_ref()) {
                 continue;
             }
 
             // Record the valid example module we found to add to the generated examples.in
             examples_in.push_str(&format!("    pub mod {};\n", example_str));
 
-            // The "switch" example name would conflict with JavaScript's switch statement. So we
-            // rename it here to switch_demo.
-            let js_entry_fn_name = if &example_str == "switch" {
-                "switch_demo".to_string()
-            } else {
-                example_str.to_string()
-            };
+            let js_entry_fn_name = example_str.to_string();
```

Comment:
Without the special case we could just pass `example_str` to the template directly.